### DISCUSSION TITLE
Private Messages in Chat

### DISCUSSION THREAD

**Discussion by JaminShanti** [jamulussoftware/jamulus#2972] [Ideas]
This would be a nice feature add.
Nothing too complex. 
The message only appears if the usersID matches the one you sent it too.
Many times I want to DM others constructive criticism but feel it would be considered rude to say it to the entire group.

**Comment by pljones** [comment]
Why not use a separate dedicated chat system?

**Comment by ann0see** [comment]
It's certainly possible and I see that it is useful sometimes (send phone number or similar). The question would be how to manage the UI.

**Comment by anprdev** [comment]
Sharing some information with other Jammers in private can be really nice. I agree with sharing Phone Numbers or Mail addresses has not to go in the Group Chat if the Message is for 1 person..

**Reply by rdica** [reply]
Dedicated chat systems provide more security for passing PII (personally identifying information), aren't as ripe for abuse, and are generally opt-in. An unencrypted, no choice to opt-in/out for priv msgs in jamulus invites unwanted/unsolicited msgs or abusive behaviour. There has already been reported abuse in chat by others, I too have seen this behaviour.
I agree with @pljones.

**Reply by anprdev** [reply]
If you see it that way I agree with you.
Abuse its on his way when we have this.
Thanks for pointing me out.
